pipenv/project.py

```diff
@@ -49,14 +50,18 @@ def _normalized(p):
     if p is None:
         return None
     loc = vistir.compat.Path(p)
-    if loc.is_absolute():
-        return normalize_drive(str(loc))
-    else:
+    if not loc.is_absolute():
         try:
             loc = loc.resolve()
         except OSError:
             loc = loc.absolute()
-        return normalize_drive(str(loc))
+    # From https://stackoverflow.com/a/35229734/5043728
+    if os.name == 'nt':
+        matches = glob.glob(re.sub(r'([^:/\\])(?=[/\\]|$)', r'[\1]', str(loc)))
```

Comment:
Can you explain what this is doing in your comment? I have no idea and i dont want to have to re-figure it out every time i look at it